src/components/teamPhotos.js

```diff
@@ -54,15 +57,63 @@ const ProfileRole = styled.span`
 
 const ProfileImg = styled(Img)`
   border-radius: 50%;
+  transition: all ease-in 150ms;
+`
+
+const ProfileImgWrapper = styled.div`
+  position: relative;
+  width: min-content;
+  margin: auto;
+
+  &:hover .profileImg {
+    filter: blur(2px) grayscale(100%);
+  }
+
+  &:hover .iconWrapper {
+    display: block;
+  }
+`
+
+const ProfileIcon = ({ dest, icon }) => {
+  return (
+    <a href={dest} target="_blank">
+      <StyledIcon icon={icon} color={SB_NAVY} />
+    </a>
+  )
+}
+
+const StyledIcon = styled(FontAwesomeIcon)`
+  color: ${props => props.color};
+  font-size: 20px;
+  margin: 0rem 0.5rem 0rem 0.5rem;
+`
+
+const IconWrapper = styled.div`
+  display: flex;
+  display: none;
+  position: absolute;
+  top: 50%;
```

Comment:
Having both `top`, `left` and `translateY`, `translateX` is a little odd to me... can you do it with only one of the two sets of rules?

Reply:
The `top` and `left` are to shift the IconWrapper element to the center of the ProfileImgWrapper (which is the center of the ProfileImg). This places the top-left corner of the IconWrapper element at the center; thus, `translateX` and `translateY` are there to shift the IconWrapper element by 50% of its own width and height to the left and up respectively. I couldn't find a way to combine them and found them necessary. Is there a better, more succinct way to achieve this?